utils/include/detray/simulation/event_generator/random_track_generator.hpp

```diff
@@ -131,8 +160,12 @@ class random_track_generator
             return *this;
         }
         DETRAY_HOST_DEVICE configuration& phi_range(scalar low, scalar high) {
-            assert(low <= high);
-            m_phi_range = {low, high};
+            auto min_phi{
```

Comment:
As mentioned above I think there should be better way to handle this

Reply:
True, will change

Reply:
Done